src/tests/address_reputation_test.ts

```diff
@@ -0,0 +1,52 @@
+import { AddressReputation } from "../coinbase/address_reputation";
+
+describe("AddressReputation", () => {
+  let addressReputation: AddressReputation;
+
+  beforeEach(() => {
+    addressReputation = new AddressReputation({
+      score: -90,
+      metadata: {
+        unique_days_active: 1,
+        total_transactions: 1,
+        token_swaps_performed: 1,
+        bridge_transactions_performed: 1,
+        smart_contract_deployments: 1,
+        longest_active_streak: 1,
+        lend_borrow_stake_transactions: 1,
+        ens_contract_interactions: 1,
+        current_active_streak: 1,
+        activity_period_days: 1,
+      },
+    });
+  });
+
+  it("returns the score", () => {
+    expect(addressReputation.score).toBe(-90);
+  });
+
+  it("returns the metadata", () => {
+    expect(addressReputation.metadata).toEqual({
+      unique_days_active: 1,
+      total_transactions: 1,
+      token_swaps_performed: 1,
+      bridge_transactions_performed: 1,
+      smart_contract_deployments: 1,
+      longest_active_streak: 1,
+      lend_borrow_stake_transactions: 1,
+      ens_contract_interactions: 1,
+      current_active_streak: 1,
+      activity_period_days: 1,
+    });
+  });
+
+  it("returns the risky as true", () => {
+    expect(addressReputation.risky).toBeTruthy();
+  });
```

Comment:
This is not a super useful test in and of itself.

Within the context of the `#risky` method we should test what is returned at `0`, below `0`, and above `0`.